src/parser/instruction.rs

```diff
@@ -465,4 +467,85 @@ mod tests {
             source: ArithmeticOperand::LiteralReal(1.0)
         }]
     );
+
+    #[test]
+    fn parse_set_scale() {
+        let tokens = lex(r#"SET-SCALE 0 "rf" 1.0; SET-SCALE 0 1 "rf" theta"#);
+        let (remainder, parsed) = parse_instructions(&tokens).unwrap();
+        let expected = vec![
+            Instruction::SetScale {
+                frame: FrameIdentifier {
+                    name: String::from("rf"),
+                    qubits: vec![Qubit::Fixed(0)],
+                },
+                scale: Expression::Number(Complex64::from(1.0)),
```

Comment:
I've defined handy `real` and `imag` macros in this crate to cut down on a little bit of verbosity when instantiating `Complex64`s. I don't they should be exported, but they're useful to keep at the crate level and use in spots like this. What do you think?

Reply:
Nice, swapped that out.